internal/schema/schemas/draft/posit-publishing-schema-v3.json

```diff
@@ -146,6 +146,12 @@
           "default": "renv",
           "description": "Package manager that will install the dependencies. If package-manager is none, dependencies will be assumed to be pre-installed on the server.",
           "examples": ["renv", "none"]
+        },
+        "requires_r_version": {
```

Comment:
The python version of this is `requires_python` with no `..._version`, WDYT about making both consistent?

Reply:
removed `_version` 👍